test/helpers/user_blocks_helper_test.rb

```diff
@@ -0,0 +1,43 @@
+# coding: utf-8
+require "test_helper"
+include ApplicationHelper
```

Comment:
It is not clear to me why it isn't included or required in UserBlocksHelper in the first place, but it fixed ruby complaing that friendly_date was undefined.